src/tools/remote/src/test/java/com/google/devtools/build/remote/worker/HttpCacheServerHandlerTest.java

```diff
@@ -0,0 +1,38 @@
+package com.google.devtools.build.remote.worker;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class HttpCacheServerHandlerTest {
```

Comment:
does this test ever run? It seems like you are missing some dependencies in the target declaration (i.e. junit).

Reply:
I don't think so (especially considering the failing `srcs_test`). I see that I need to include this test somewhere but cannot quite figure out where exactly. Would appreciate your suggestion/help :)